docs/docs/playground.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-id: prototype-an-assistant
+id: playground
```

Comment:
tests are failing, you'll need to update the `docs/sidebars.js` file 😄

Reply:
thank you :)